index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+  rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+  rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.15.1/devicon.min.css">
+ 
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <title>mobile skeleton</title>
+</head>
+<body>
```

Comment:
- [ ] @jemutakiplagat  as the previous reviewer mentioned kindly fix the overflow to prevent scrolling left and right


![overflow](https://user-images.githubusercontent.com/89282221/185391437-88a93a44-d30b-4caf-881c-05e52cc14e2d.gif)